modules/openapi-generator/src/main/resources/JavaPlayFramework/securityApiUtils.mustache

```diff
@@ -135,6 +137,7 @@ public class SecurityAPIUtils {
                 Algorithm algorithm = Algorithm.RSA256((RSAPublicKey) publicKey, null);
                 tokenVerifier = JWT.require(algorithm)
                         .withIssuer(issuer)
+                        .acceptLeeway(leeway)
```

Comment:
this is required in some circumstances to tolerate some clock synchronization issues between backends and oauth authentification server. Configurable by the application to the desired value, default 3 sec.